package-lock.json

```diff
@@ -0,0 +1,13230 @@
+{
```

Comment:
You can safely delete this file since we're using the yarn lockfile instead.

Reply:
Okay, I have removed this